pages/community/styleguide/code-examples.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Code examples
+weight: 20
+---
+
+
```

Comment:
Please remove the extra spaces of lines 5-6, we don't want huge empty spots in our docs' formatting/spacing 😂✌🏽

Reply:
done